### PR TITLE
style: run prettier on learn

### PR DESCRIPTION
**Pull request by nschonni**
This was run with embeddedLanguageFormatting off to reduce noise

### CODE REVIEW

**Comment by nschonni** [comment]
Prettier is removing the escaping for tags inside of alt text for images causing Markdownlint issues. I think prettier is correct, so I've flipped this to draft and raised https://github.com/DavidAnson/markdownlint/issues/579

**Comment by nschonni** [comment]
I re-escaped the tags to make Markdownlint happy. There were one or 2 other minor prettier ones I ran into later that might need workarounds, so this doesn't need to wait for an upstream fix

File: files/en-us/learn/html/introduction_to_html/getting_started/index.md [hunk 1]
```diff
@@ -263,7 +263,7 @@ This would output the following:
 
 Elements can also have attributes. Attributes look like this:
 
-![&amp;amp;amp;amp;amp;lt;p class="editor-note">My cat is very grumpy&amp;amp;amp;amp;amp;lt;/p>](grumpy-cat-attribute-small.png)
+![&amp;amp;amp;amp;lt;p class="editor-note">My cat is very grumpy&amp;amp;amp;amp;lt;/p>](grumpy-cat-attribute-small.png)
```
Comment (teoli2003) [inline files/en-us/learn/html/introduction_to_html/getting_started/index.md:266]: 1. I don't understand this _alt_ text (we probably want to update it to something better).
2. I don't understand why Prettier does this correction!

File: files/en-us/learn/html/introduction_to_html/the_head_metadata_in_html/index.md [hunk 2]
```diff
@@ -77,12 +77,12 @@ We've already seen the {{htmlelement("title")}} element in action — this can b
...
-    2. Press the "Raw" button on the GitHub page, which causes the raw code to appear (possibly in a new browser tab). Next, choose your browser's _Save Page As…_ menu and choose a sensible place to save the file.
+   1. Copy and paste the code out of the page and into a new text file in your code editor, then save it in a sensible place.
+   2. Press the "Raw" button on the GitHub page, which causes the raw code to appear (possibly in a new browser tab). Next, choose your browser's _Save Page As…_ menu and choose a sensible place to save the file.
 
 2. Now open the file in your browser. You should see something like this:
 
-    ![A simple web page with the title set to \<title> element, and the \<h1> set to \<h1> element.](title-example.png)It should now be completely obvious where the `<h1>` content appears, and where the `<title>` content appears!
+   ![A simple web page with the title set to \<title> element, and the \<h1> set to \<h1> element.](title-example.png)It should now be completely obvious where the `<h1>` content appears, and where the `<title>` content appears!
```
Comment (teoli2003) [inline files/en-us/learn/html/introduction_to_html/the_head_metadata_in_html/index.md:85]: Side-fix
```suggestion
   ![A simple web page with the title set to \<title> element, and the \<h1> set to \<h1> element.](title-example.png)
   
   It should now be completely obvious where the `<h1>` content appears and where the `<title>` content appears!
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]

File: files/en-us/learn/html/introduction_to_html/getting_started/index.md (hunk 1 above)
Comment (nschonni) [reply files/en-us/learn/html/introduction_to_html/getting_started/index.md]: I tried updating the alt text. I think something previously was escaping the `&`, but Prettier is slowly unescaping them again.

**Review by teoli2003 — APPROVED** [review]

**Pull request merged**